### PR TITLE
compat: Enable & start camera_service

### PR DESCRIPTION
**Pull request by fredldotme**

### CODE REVIEW

File: compat/media/camera_service.cpp [hunk 1]
```diff
@@ -48,9 +46,7 @@ int main(int argc, char** argv)
     // for creating a new IMediaRecorder (MediaRecorder) instance over Binder
     MediaRecorderFactory::instantiate();
     // Enable audio recording for camera recording
-#if ANDROID_VERSION_MAJOR==5 && WANT_UBUNTU_CAMERA_HEADERS
     CameraRecordService::instantiate();
-#endif
     CameraService::instantiate();
```
Comment (NotKit) [inline compat/media/camera_service.cpp:51]: Have you checked it does not conflict with minimediaservice? I left only CameraRecordService running on Halium 9 (see https://github.com/Halium/libhybris/commit/a76968e17894836e93b90340409e78048a70ec32).

Comment (fredldotme) [reply compat/media/camera_service.cpp:51]: You mean CameraService conflicting with minimediaservice? I have both processes running without issues but will happily test your additional `#ifdef` on 7.1.

Comment (fredldotme) [reply compat/media/camera_service.cpp:51]: Done, doesn't seem to be necessary.

**Comment by fredldotme** [comment]
If there are no objections to it I would like to merge this within 48 hours.

**Pull request merged**